MoreLinq/ScanBy.cs

```diff
@@ -40,14 +40,17 @@ static partial class MoreEnumerable
         /// <returns>
         /// A sequence of keys paired with intermediate accumulator states.
         /// </returns>
-
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="keySelector"/> is null</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="seedSelector"/> is null</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="accumulator"/> is null</exception>
         public static IEnumerable<KeyValuePair<TKey, TState>> ScanBy<TSource, TKey, TState>(
             this IEnumerable<TSource> source,
             Func<TSource, TKey> keySelector,
             Func<TKey, TState> seedSelector,
             Func<TState, TKey, TSource, TState> accumulator)
         {
-            return source.ScanBy(keySelector, seedSelector, accumulator, null);
+            return source.ScanBy(keySelector, seedSelector, accumulator, comparer: null);
```

Comment:
Is the argument name being added for clarity?

Reply:
Yes.